src/oak.py

```diff
@@ -94,7 +93,10 @@ class Properties(NamedTuple):
         supports_pcd: bool = True
         """Whether the camera has a valid implementation of ``get_point_cloud``"""
 
-    MODEL: ClassVar[Model] = Model(ModelFamily("viam", "camera"), "oak-d")
+    MODELS: ClassVar[Tuple[Model]] = (
+        Model(ModelFamily("viam", "camera"), "oak"),
+        Model(ModelFamily("viam", "camera"), "oak-ffc"),
```

Comment:
Wasn't there another model that this supports?

Reply:
Right sorry for not documenting this ticket well. There are 3 models we would like to support `oak`, `oak-d`, and `oak-ffc`. However, the [oak-d module](https://app.viam.com/module/viam/oak-d) currently has ownership of the `oak-d` model. After https://github.com/viamrobotics/app/pull/3684 is in, we can migrate users to this new module, then use the UI to delete the module. I checked-- there is one external user using the old oak-d module.

Reply:
I can add the `oak-d` model into the new OAK module after we're cleared to deprecate and delete the oak-d module

Reply:
I think for the sake of not breaking users:
1. Don't change the *module* name, keep that as "oak-d" int he metas json. The rest of the changes can be applied as long as that doesn't change.
2. Add the models we support here. 
3. In a separate PR update the *module* name, make sure you release it and migrate at the same time as an external user might want it. Check with sol-eng I think I know the user.

Reply:
Notes from 1:1 regarding this:
- Pinged registry folks about why models are uniquely maintained at the org scope vs. the module scope (what it should be)
- Keep meta.json tied to new module `oak`
- Bothering Fleet/Michael Lee about manually updating the config of the one external user still on the `oak-d` module. Afterwards we will just delete that module.